src/EnergyPlus/UnitVentilator.hh

```diff
@@ -67,127 +67,143 @@ struct EnergyPlusData;
 
 namespace UnitVentilator {
 
+    enum class CoilOption
+    {
+        Invalid = -1,
+        None,
```

Comment:
Is `None` the same thing as `Invalid`? Strange to think it would be a legitimate runtime option.

Reply:
A13, \field Coil Option
        \required-field
        \type choice
        \key None
        \key Heating
        \key Cooling
        \key HeatingAndCooling

Reply:
🤯 What is "None" coil? Asking for a friend.

Reply:
Legacy code for an input that is redundant. The model knows what coils are used, or not. Tell your friend I could remove that entirely but it will take some time to work through the ToDo list. I'll check what the model does when no coil exists and think about removing that input.

Reply:
My friend (also named Amir coincidentally) says its fine to put this off for now if it's not a trivial fix.